### DISCUSSION TITLE
Game Server LXC

### DISCUSSION THREAD

**Discussion by Sedation6612** [community-scripts/ProxmoxVE#776] [Ideas]
I think having a section for creating game servers in LXC would be a great addition, can also bring in a new wave of users as well.

I’m aware some game servers require complex configuration, but some are simple, example, Terraria.

What do you all think?

**Comment by michelroegl-brunner** [comment]
There are currently no plans to include game server scripts at the moment, due to the afore mentioned complexity as well es the high maintenance demand.
